### PR TITLE
Feature/enable polygon

### PR DESCRIPTION
**Pull request by PorlyBe**

### CODE REVIEW

File: src/components/Loading/Loading.tsx [hunk 1]
```diff
@@ -16,6 +16,7 @@ import { Button } from "../Button";
 import { useWeb3Modal } from "@web3modal/wagmi/react";
 import connectWallet from "../../assets/connect-wallet.png";
 import halfMoon from "../../assets/half-moon.png";
+import halfMoonPollygon from "../../assets/half-moon-polygon.png";
```
Comment (micahpatng) [inline src/components/Loading/Loading.tsx:19]: polygon lol

**Review by micahpatng — APPROVED** [review]
let's gooo

File: src/components/Loading/Loading.tsx (hunk 1 above)
Comment (PorlyBe) [reply src/components/Loading/Loading.tsx:19]: i swear my autocorrect does this

Comment (PorlyBe) [reply src/components/Loading/Loading.tsx:19]: merging anyways... will fix once we introduce styled themes

**Pull request merged**